nodes/locales/en-US/clarify_insert.html

```diff
@@ -14,22 +14,22 @@ <h3>General Properties</h3>
   <h3>Inputs</h3>
   <dl class="message-properties">
     <dt>ID <span class="property-type">msg.payload.id</span></dt>
-    <dd>A unique id per signal that has to match the regex for Clarify ID's <code>/^[a-z0-9_-]{1,40}$/</code>. If the id is changed will a new signal be created.</dd>
+    <dd>A unique id per signal that has to match the regex for the Clarify Input ID, <code>/^[a-z0-9_-]{1,40}$/</code>. If the id is changed will a new signal be created.</dd>
     <dd>Expects: string</dd>
 
     <dt>Name <span class="property-type">msg.payload.name</span></dt>
     <dd>The name used for the signal. Defaults to the signal ID.</dd>
     <dd>Expects: string</dd>
 
-    <dt class="optional">Data type <span class="property-type">msg.payload.dataType</span></dt>
-    <dd>The data type for the signal. Must be set as msg.payload.dataType and be either numeric or enum. Defaults to numeric.</dd>
+    <dt class="optional">Type <span class="property-type">msg.payload.type</span></dt>
+    <dd>The type for the signal. Must be set as msg.payload.type and be either numeric or enum. Defaults to numeric.</dd>
```

Comment:
Need to change this (in separate PR).

```suggestion
    <dt class="optional">type <span class="property-type">string</span></dt>
    <dd>The type for the signal can be  <span class="property-type">"numeric"</span> or<span class="property-type"> "enum"</span>. Fetched from msg.payload.type by default.</dd>
```